src/dreyfus_fabric_cleanup.erl

```diff
@@ -53,7 +53,7 @@ cleanup_local_purge_doc(DbName, ActiveSigs) ->
     end, [], LocalShards),
 
     DeadDirs = DirList -- ActiveDirs,
-    lists:foldl(fun(IdxDir) ->
+    lists:foreach(fun(IdxDir) ->
         Sig = dreyfus_util:get_signature_from_idxdir(IdxDir),
         case Sig of undefined -> ok; _ ->
             DocId = dreyfus_util:get_local_purge_doc_id(Sig),
```

Comment:
The internal foldl could also be a foreach

Reply:
changed in 597db54

Reply:
Sorry I mean this one `lists:foldl(fun(LS, _AccOuter) ->` could also be a foreach

Reply:
thanks again @garrensmith 
i fixed in 85b5612, and can you please check again?